### ISSUE TITLE
FileNotFound Error

### ISSUE THREAD

**Issue by ToxicZombies** [fifty-six/zig.SteamManifestPatcher#3]
As with another user I have the file not found error, it says as follows:

error.PatternNotFound
Unable to dump stack trace: FileNotFound
Press ENTER to close.

Also Win10, however when I ran as admin it did not change anything

**Comment by fifty-six** [comment]
Does [this build](https://github.com/fifty-six/zig.SteamManifestPatcher/suites/3307408713/artifacts/77359333) work for you?

**Comment by ToxicZombies** [comment]
error.PatternNotFound
Unable to dump stack trace: FileNotFound
Press ENTER to close.

So no. Sorry.

**Comment by Varow** [comment]
I just had this issue myself, however, I have Steam set to "Run as Administrator" by default in my Compatibility settings (the OP might possibly as well). If that is the case, you'll just need to simply also run the `SteamDepotDownpatcher.exe` as an administrator by right-clicking file and then hitting "Run as Administrator".

**Comment by fifty-six** [comment]
You can get a more in-depth stacktrace if you run the exe with the pdb next to it, like it's set up in the zip from CI. Note that if you've ran the exe once on subsequent tries it'll fail as it has already patched the application, so I'd restart steam before trying it. What version of steam are you on? I wasn't able to reproduce this unfortunately.

**Comment by ToxicZombies** [comment]
Yeah as I said in my first post "Run as Admin" didn't do anything. I do not know where to find the build number for steam.

**Comment by fifty-six** [comment]
Can you try this build? [bin.zip](https://github.com/fifty-six/zig.SteamManifestPatcher/suites/3318020284/artifacts/77707761)
  
Does `download_depot` work in steam anyways? Also, did you restart the steam client before re-trying? 

----
This is where you can find the steam version:
![image](https://user-images.githubusercontent.com/10470872/126851645-1e029fbb-f5de-4a8f-b124-ca7a4284f340.png)

**Comment by ToxicZombies** [comment]
Oh. I got it to work.. Yeah I forgot to ACTUALLY restart steam, I just closed and reopened it, which it only occurred to me now that it stayed open in the background so it never tried to update itself....... I didn't try the build you just sent, and still have not checked my steam version, however I will remember that PDF in the future if I need it. I am currently confused on what you mean by download_depot however. But my problem has been fixed here, thank you all very much!
